homeassistant/components/mystrom/light.py

```diff
@@ -121,6 +138,16 @@ def is_on(self):
         """Return true if light is on."""
         return self._state
 
+    @property
+    def device_info(self) -> DeviceInfo:
+        """Return the device info for the light entity."""
+        return DeviceInfo(
+            identifiers={(DOMAIN, self._name)},
```

Comment:
This is not a valid identifier.

Reply:
I now use the mac address. Would this work?